picard/util/filenaming.py

```diff
@@ -326,14 +327,21 @@ def _get_filename_limit(target):
         limit = limits[target]
     except KeyError:
         # we need to call statvfs on an existing target
-        d = target
```

Comment:
There's a comment just before the start of this function about Python statvfs and f_fsid not being supported though it changed in version 3.7: Added the f_fsid attribute.

I wonder if the suggested merge would make sense now.

Reply:
I'll look into it